graphics/diff-pdf/Portfile

```
@@ -28,18 +28,17 @@ long_description \
 
 homepage            https://vslavik.github.io/diff-pdf
 
-checksums           rmd160  a43d407ec3e11818062e979d685d668155d9f29d \
-                    sha256  0eb81af6b06593488acdc5924a199f74fe3df6ecf2a0f1be208823c021682686 \
-                    size    137715
+checksums           rmd160  ee2e4302130a6db08cf032d80155dfd757dbea72 \
+                    sha256  e7b8414ed68c838ddf6269d11abccdb1085d73aa08299c287a374d93041f172e \
+                    size    135755
 
 depends_lib         path:lib/libcairo.dylib:cairo \
                     port:poppler \
-                    port:${wxWidgets.port}
+                    port:${wxWidgets.port} \
+                    port:glib2
```

@srirangav If glib2 is needed, then the glib2 dependency should be written so that glib2-devel could satisfy it.

```suggestion
                    path:lib/pkgconfig/glib-2.0.pc:glib2
```

We typically sort dependencies alphabetically. I'll fix it.